frontend/public/components/deployment.tsx

```diff
@@ -217,25 +226,40 @@ const ReplicaSetsTab: React.FC<ReplicaSetsTabProps> = ({ obj }) => {
 };
 
 const { details, editYaml, pods, envEditor, events } = navFactory;
-export const DeploymentsDetailsPage: React.FC<DeploymentsDetailsPageProps> = (props) => (
-  <DetailsPage
-    {...props}
-    kind={deploymentsReference}
-    menuActions={menuActions}
-    pages={[
-      details(DeploymentDetails),
-      editYaml(),
-      {
-        href: 'replicasets',
-        name: 'Replica Sets',
-        component: ReplicaSetsTab,
-      },
-      pods(),
-      envEditor(environmentComponent),
-      events(ResourceEventStream),
-    ]}
-  />
-);
+export const DeploymentsDetailsPage: React.FC<DeploymentsDetailsPageProps> = (props) => {
+  const { ns } = props.match?.params;
+  const csvResource = React.useMemo(
+    () => ({
+      isList: true,
+      kind: referenceForModel(ClusterServiceVersionModel),
+      namespace: ns,
+      optional: true,
+    }),
```

Comment:
nit: there is not much benefit but can put this resource to be fetched in utils as used in few other places or a customHook `useCsvWatchResource` which would return `csvData` and take `ns`